recipes/knit/build.sh

```diff
@@ -0,0 +1,3 @@
+export JAVA_HOME=$PREFIX
+java -version
+$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
```

Comment:
FYI normally we do `--record=record.txt`, which drops it in the working directory. That whole directory will be removed by `conda-build` as long as the build completes. Though can see no harm in it going to `/tmp` either.